wandb/util.py

```diff
@@ -755,9 +758,12 @@ def request_with_retry(func, *args, **kwargs):
             ):
                 logger.info("Rate limit exceeded, retrying in %s seconds" % delay)
             else:
+                pass
                 logger.warning(
-                    "requests_with_retry encountered retryable exception: %s. args: %s, kwargs: %s",
+                    "requests_with_retry encountered retryable exception: %s. func: %s, response: %s, args: %s, kwargs: %s",
                     e,
+                    func,
+                    e.response.content,
```

Comment:
I don't think we always have `e.response`. 
In my case  request is throwing `urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))`
Which is a retryable exception but `e.response` is not set , making a `e.response.content` and `AttributeError`.

Reply:
Thanks for this investigation!  We'll take a look.